lib/segment/Cargo.toml

```diff
@@ -55,6 +55,7 @@ validator = { version = "0.16", features = ["derive"] }
 chrono = { version = "0.4.26", features = ["serde"] }
 
 sysinfo = "0.29"
+charabia = "0.7.2"
```

Comment:
right, I was also thinking that this might help with situations like windows build fails, potentially. If we want to focus only on CJK it makes sense to disable others.

Reply:
OK, so I've measured the binary size on fresh `dev` and here. Results doesn't look great.. 

```
| setup          | commit   | size  |
|----------------+----------+-------|
| dev            | edf3d88e | 52.3M |
| charabia: full | 68266067 | 92.7M |
| charabia: cjk  | --       | 92.7M |

```
Linux(NixOS), x86_64

These are the sizes of `target/release/qdrant` after executing `cargo build --release`.

I've also expected that removing features from charabia would exclude some crates from deps and the size will be reduced. However, -- I've checked couple of times and did a cleanup just to make sure -- the size was the same as with full charabia. 

This is the command I've used to limit features:
```bash
cargo add charabia --no-default-features --features "chinese,japanese,korean"
    Updating crates.io index
      Adding charabia v0.7.2 to dependencies.
             Features:
             + chinese
             + japanese
             + korean
             - greek
             - hebrew
             - japanese-transliteration
             - latin-camelcase
             - lindera
             - thai
```

It feels too much for one type of tokenizer to take half of binary size

Reply:
it looks like meilisearch somehow fits 50mb

Reply:
It's not too surprising that the CJK tokenizers take so much space, as they each include a full dictionary for their respective language.